src/app/view/paperView.js

```diff
@@ -132,8 +132,48 @@ export default class PaperView {
             layer.children[i].selected = true;
         }
     }
+
     /**
-     * Converts the layers to SVG format 
+     * Center all the components on the canvas
+     * @returns {void}
+     * @memberof PaperView
+     */
+    centerAll() {
+        let layer = this.paperLayers[this.activeLayer];
+        if (layer.children.length > 1) {
+            // Find the center of the design
+            let leftBound = layer.children[0].position.x;
+            let rightBound = layer.children[0].position.x;
+            let upperBound = layer.children[0].position.y;
+            let lowerBound = layer.children[0].position.y;
+            for (var i = 0; i < layer.children.length -1; i++) {
+                let component = layer.children[i].position;
+                if (component.x < leftBound) {
+                    leftBound = component.x;
+                } else if (component.x > rightBound) {
+                    rightBound = component.x;
+                }
+                if (component.y < upperBound) {
+                    upperBound = component.y;
+                } else if (component.y > lowerBound) {
+                    lowerBound = component.y;
```

Comment:
While this would work visually, it would not update the `Component` and `Connection` objects in the design.